### PR TITLE
feat/#60

### PR DESCRIPTION
**Pull request by DungNguyen2003**

### CODE REVIEW

**Comment by nc-minh** [comment]
@DungNguyen2003 pls resolve conflict.

**Comment by DungNguyen2003** [comment]
done _-_

File: src/app/containers/TestShow/index.tsx [hunk 1]
```diff
@@ -1,20 +1,66 @@
...
+  }, [openPropup]);
+
+  const handleChangeExamDuration = useCallback(
+    (e: React.ChangeEvent<HTMLInputElement>) => {
+      setExamDuration(Number(e.target.value));
+      if (durationError !== "") {
+        setDurationError("");
+      }
+    },
+    [examDuration],
```
Comment (nc-minh) [inline src/app/containers/TestShow/index.tsx:43]: missing durationError in dependencies

File: src/app/containers/TestShow/index.tsx [hunk 2]
```diff
@@ -1,20 +1,66 @@
...
+        duration: examDuration * 60000,
+        exam_id: exam?._id!,
+      });
+      setExamDuration(0);
+      handleClosePopup();
+      setDurationError("");
+    } catch (error) {
+      setDurationError("Có lỗi xảy ra vui lòng liên hệ bộ phận phát triển");
+    }
+  }, [examDuration]);
```
Comment (nc-minh) [inline src/app/containers/TestShow/index.tsx:62]: missing some dependencies

File: src/app/components/PopupDuration/index.tsx [hunk 3]
```diff
@@ -0,0 +1,89 @@
+import { LoadingButton } from "@mui/lab";
```
Comment (nc-minh) [inline src/app/components/PopupDuration/index.tsx:1]: Why do you need to create this component? If you want to create a custom popup it has already been defined in **_ModalCustomization_** component

**Review by nc-minh — CHANGES_REQUESTED** [review]

**Comment by DungNguyen2003** [comment]
fixed =_=

**Review by nc-minh — APPROVED** [review]

**Pull request merged**